service/src/java/org/apache/hive/service/cli/operation/SQLOperation.java

```diff
@@ -328,7 +328,8 @@ public Object run() throws HiveSQLException {
           if (!embedded) {
             LogUtils.registerLoggingContext(queryState.getConf());
           }
-          ShimLoader.getHadoopShims().setHadoopQueryContext(queryState.getQueryId());
+          ShimLoader.getHadoopShims()
+              .setHadoopQueryContext(queryState.getQueryId() + " User:" + parentSessionState.getUserName());
```

Comment:
For consistency with the other call sites setting query context, please add a space after the colon:

```
... + " User: " + ...`
```

(However, also see my other comment about whether or not we should use spaces. Whatever is decided for the format should be consistent at all call sites.)

Reply:
I removed the space itself,